pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -183,22 +182,6 @@ func (opt *Options) Run(suite *TestSuite, junitSuiteName string) error {
 		}
 	}
 
-	// This ensures that tests in the identified paths do not run in parallel, because
-	// the test suite reuses shared resources without considering whether another test
-	// could be running at the same time. While these are technically [Serial], ginkgo
-	// parallel mode provides this guarantee. Doing this for all suites would be too
-	// slow.
-	setTestExclusion(tests, func(suitePath string, t *testCase) bool {
```

Comment:
where did this move and if straight removed, how did you determine this was now safe?

Reply:
@soltysh ?  From what I can tell I didn't have this dropped in my ginkgo v2 rebase